OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -395,7 +395,8 @@ public String getOAuthClientId() {
     }
 
     public String getOrganizationCode() {
-        return getString(ORGANIZATION_CODE, "");
+        String orgCode = getString(ORGANIZATION_CODE, "");
+        return orgCode.length() > 0 ? orgCode : null;
```

Comment:
Yeah, I think this can be switched to use the default method. Whether an empty string should be interpreted as an indicator to skip the query parameter should be a decision left to the server.